extensions/wikia/AdEngine/js/lookup/prebid/adapters/veles.js

```diff
@@ -67,50 +67,80 @@ define('ext.wikia.adEngine.lookup.prebid.adapters.veles', [
 		}
 	}
 
+	function getPriceFromTitle(responseXML) {
+		var lineItemTitle = responseXML.documentElement.querySelector('AdTitle');
+
+		return lineItemTitle ? getPriceFromString(lineItemTitle.textContent) : null;
+	}
+
+	function getPriceFromString(title) {
+		var re = new RegExp('ve(\[0-9]{4})(xx|ic|lb)', 'i'),
+			results = re.exec(title);
+
+		return results && results[1] ? parseInt(results[1], 10) / 100 : null;
+	}
+
+	function getPriceFromConfigId(ad) {
+		var adConfigPrice,
+			id = ad.getAttribute('id');
+
+		if (id) {
+			adConfigPrice = instantGlobals.wgAdDriverVelesBidderConfig[id];
+			if (adConfigPrice) {
+				return parseInt(adConfigPrice, 10) / 100;
+			}
+		}
+	}
+
+	function getPriceFromConfigAdSystem(ad) {
+		var adConfigPrice = instantGlobals.wgAdDriverVelesBidderConfig[adxAdSystem],
+			adSystem;
+
+		if (adConfigPrice) {
+			adSystem = ad.querySelector('AdSystem');
+			if (adSystem && adSystem.textContent === adxAdSystem) {
+				return parseInt(adConfigPrice, 10) / 100;
+			}
+		}
+	}
+
 	function fetchPrice(vastRequest) {
 		var ad,
-			adConfigPrice,
-			adParameters,
-			adSystem,
 			parameters,
+			price,
 			responseXML = vastRequest.responseXML;
 
 		if (!responseXML) {
 			return 0;
 		}
 
 		ad = responseXML.documentElement.querySelector('Ad');
-		if (ad && instantGlobals.wgAdDriverVelesBidderConfig) {
-			if (ad.getAttribute('id')) {
-				adConfigPrice = instantGlobals.wgAdDriverVelesBidderConfig[ad.getAttribute('id')];
-				if (adConfigPrice) {
-					return parseInt(adConfigPrice, 10) / 100;
+
+		if (ad) {
+			price = getPriceFromTitle(responseXML);
+
+			if (!price && instantGlobals.wgAdDriverVelesBidderConfig) {
+				price = getPriceFromConfigId(ad);
+
+				if (!price) {
+					price = getPriceFromConfigAdSystem(ad);
 				}
 			}
 
-			adConfigPrice = instantGlobals.wgAdDriverVelesBidderConfig[adxAdSystem];
-			if (adConfigPrice) {
-				adSystem = ad.querySelector('AdSystem');
-				if (adSystem && adSystem.textContent === adxAdSystem) {
-					return parseInt(adConfigPrice, 10) / 100;
-				}
+			if (geo.isProperGeo(instantGlobals.wgAdDriverVelesVastLoggerCountries)) {
+				logVast(vastRequest);
 			}
 		}
 
-		adParameters = responseXML.documentElement.querySelector('AdParameters');
-		if (adParameters) {
-			parameters = parseParameters(adParameters);
+		if (!price) {
+			parameters = parseParameters(responseXML.documentElement.querySelector('AdParameters'));
 
 			if (parameters.veles) {
-				return parseInt(parameters.veles, 10) / 100;
+				price = parseInt(parameters.veles, 10) / 100;
 			}
 		}
 
-		if (ad && geo.isProperGeo(instantGlobals.wgAdDriverVelesVastLoggerCountries)) {
-			logVast(vastRequest);
```

Comment:
The order and returns were important in order to prevent logging of "known" VAST xmls. After those changes  VAST with AdParameters/price with title/configured ID or AdSystem will be logged to Kibana (in general - all xmls) if I understand it correctly